core/src/main/java/io/grpc/internal/ClientCallImpl.java

```diff
@@ -157,13 +157,14 @@ public void start(Listener<RespT> observer, Metadata headers) {
 
     ClientStreamListener listener = new ClientStreamListenerImpl(observer);
     ListenableFuture<ClientTransport> transportFuture = clientTransportProvider.get(callOptions);
+
     if (transportFuture.isDone()) {
```

Comment:
It was earlier in this PR, but since it was moved back into CCI it is not.  (same as prior to this PR existing).